### PR TITLE
Add some links from jobs.cz team

### PR DESCRIPTION
**Pull request by jirinovak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,4 +3,38 @@
...
-To Be Done
+- [Development](#development)
+- [Core review](#code-review)
+- [Q&A](#qa)
+- [Git](#git)
+- [Books](#books)
+- [Newsletter](#newsletters)
+
+## Development
+- [@Ocramius - Fluent interfaces are evil](https://ocramius.github.io/blog/fluent-interfaces-are-evil/)
```
Comment (OndraM) [inline README.md]: What about swap author and article title? The title is more important imo. (The same below)

File: README.md [hunk 2]
```diff
@@ -3,4 +3,38 @@
...
+- Matthias Noback - article series
+    - https://www.ibuildings.nl/blog/2016/01/programming-guidelines-php-developers-part-1-reducing-complexity
+    - https://www.ibuildings.nl/blog/2016/01/programming-guidelines-php-developers-part-2-getting-rid-null
+    - https://www.ibuildings.nl/blog/2016/08/why-write-tests
+- [Symfony komponenty](https://www.zdrojak.cz/serialy/symfony-po-kruckach/)
```
Comment (OndraM) [inline README.md]: I'd create somethinglike "PHP specific" subsection as part of this section and put this Symfony topic here (And I think more will come). The other links are language-independent.

File: README.md [hunk 3]
```diff
@@ -3,4 +3,38 @@
 A curated list of amazingly awesome resources for (not only) LMC developers.
 
 ## Table of Contents
-To Be Done
+- [Development](#development)
+- [Core review](#code-review)
```
Comment (OndraM) [inline README.md]: typo, s/Core/Code/

File: README.md [hunk 4]
```diff
@@ -3,4 +3,38 @@
...
+- [Q&A](#qa)
+- [Git](#git)
+- [Books](#books)
+- [Newsletter](#newsletters)
+
+## Development
+- [@Ocramius - Fluent interfaces are evil](https://ocramius.github.io/blog/fluent-interfaces-are-evil/)
+- Matthias Noback - article series
+    - https://www.ibuildings.nl/blog/2016/01/programming-guidelines-php-developers-part-1-reducing-complexity
```
Comment (OndraM) [inline README.md]: Links are broken, also we can show here their title via markdown, not just the URL.

Proper links:
- https://www.ibuildings.nl/blog/2016/01/programming-guidelines-part-1-reducing-complexity
- https://www.ibuildings.nl/blog/2016/01/programming-guidelines-part-2-getting-rid-null
- Third one is OK :)

File: README.md [hunk 5]
```diff
@@ -3,4 +3,38 @@
 A curated list of amazingly awesome resources for (not only) LMC developers.
 
 ## Table of Contents
-To Be Done
+- [Development](#development)
+- [Core review](#code-review)
+- [Q&A](#qa)
+- [Git](#git)
+- [Books](#books)
+- [Newsletter](#newsletters)
```
Comment (OndraM) [inline README.md]: Newsletters

File: README.md [hunk 6]
```diff
@@ -3,4 +3,38 @@
...
+## Code review
+- [Code reviews v praxi](https://www.zdrojak.cz/clanky/code-reviews-praxi/])
+- [Code review checklist](https://www.zdrojak.cz/clanky/code-review-checklist/)
+- [Co nás naučilo code review](https://www.zdrojak.cz/clanky/co-nas-naucilo-code-review/)
+
+## Q&A
+- [@OndraM - Funkční systémové testování: když unit-testy nestačí](http://prvnivcesku.cz/systemove-testovani/)
+
+## Git
+- [Jak a proč fixupovat](https://filip-prochazka.com/blog/git-fixup )
```
Comment (OndraM) [inline README.md]: unnecessary space

**Review by OndraM — CHANGES_REQUESTED** [review]

**Review by OndraM — APPROVED** [review]

**Review by legendik — APPROVED** [review]

**Pull request merged**